KesaApplication/ViewController.swift

```diff
@@ -8,18 +8,180 @@
+            firstNameLabel.textColor = UIColor.blackColor()
+            lastNameLabel.textColor = UIColor.blackColor()
+            emailLabel.textColor = UIColor.orangeColor()
+            passwordLabel.textColor = UIColor.blackColor()
+        }
 
+        else {
+            firstNameLabel.textColor = UIColor.blackColor()
+            lastNameLabel.textColor = UIColor.blackColor()
+            emailLabel.textColor = UIColor.blackColor()
+            passwordLabel.textColor = UIColor.orangeColor()
+        }
+        
+    }
+    
+    
+    @IBAction func onCreateAccountButtonClickListener(sender: UIButton) {
+        
+        let utility = Utility() // An instance of the Utility Class
+        
+        let userFirstName = firstName.text
+        let userLastName = lastName.text
+        let userEmail = email.text
+        let userPassword = password.text
+        
+        // Booleans to check if the names have numbers included in them 
+        let checkFirstName = utility.isValidNameFormat(userFirstName!)
+        let checkLastName = utility.isValidNameFormat(userLastName!)
+        
+        /* Check for empty fields */
+        if (userFirstName!.isEmpty) {
+            
+            // Display alert message
+            displayAlertMessage ("First name field is required");
+            
+            return
+        }
+        
+        
+        else if (userLastName!.isEmpty) {
+         
+            displayAlertMessage("Last name field is required")
+            
+            return
+        }
+            
+        else if (userEmail!.isEmpty) {
+            
+            displayAlertMessage("Email field is required")
+            
+            return
+        }
+            
+        else if (userPassword!.isEmpty) {
+            
+            displayAlertMessage("Password field is required")
+            
+            return
+        }
+        
+        // check if the names are valid (no numbers included)
+        else if (!checkFirstName) {
+            
+            displayAlertMessage("Invalid first name")
+            
+            return
+        }
+        
+        else if (!checkLastName) {
+            
+            displayAlertMessage("Invalid last name")
+            
+            return
+        }
+        
+        else { // Email validity check
+            
+            if utility.isValidEmailFormat(userEmail!)  {
+            
+            
+                // Store data and etc
+            
+            }
+            
+            else {
+                displayAlertMessage ("Invalid email address")
+            
+                return
+            }
+        }
+    }
+ 
+    
+    /* Function for displaying alert messages */
+    func displayAlertMessage (userMessage: String){
+        
+        let alert = UIAlertController(title: "Alert", message: userMessage,
+                                      preferredStyle:
+                                      UIAlertControllerStyle.Alert)
+        
```

Comment:
Don't put unnecessary line in between. Only use the extra lines to separate the logics.